src/pages/home/report/ReportActionItemMessage.tsx

```diff
@@ -120,14 +123,34 @@ function ReportActionItemMessage({action, transaction, displayAsGroup, reportID,
 
         // Approving or submitting reports in oldDot results in system messages made up of multiple fragments of `TEXT` type
         // which we need to wrap in `<Text>` to prevent them rendering on separate lines.
-
         return shouldWrapInText ? <Text style={styles.ltr}>{reportActionItemFragments}</Text> : reportActionItemFragments;
     };
 
     return (
         <View style={[styles.chatItemMessage, style]}>
             {!isHidden ? (
-                renderReportActionItemFragments(isApprovedOrSubmittedReportAction)
+                <>
+                    {renderReportActionItemFragments(isApprovedOrSubmittedReportAction)}
+                    {action.actionName === CONST.REPORT.ACTIONS.TYPE.IOU && ReportUtils.hasMissingInvoiceBankAccount(reportID) && (
+                        <Button
+                            style={[styles.mt2, styles.alignSelfStart]}
+                            medium
+                            success
+                            text={translate('workspace.invoices.paymentMethods.addBankAccount')}
+                            onPress={() => {
+                                const policyID = ReportUtils.getReport(reportID)?.policyID;
+
+                                if (!policyID) {
+                                    return;
+                                }
+
+                                // TODO: Uncomment the following line when the invoices screen is ready - https://github.com/Expensify/App/issues/45175.
+                                // Navigation.navigate(ROUTES.WORKSPACE_INVOICES.getRoute(policyID))
+                                Navigation.navigate(ROUTES.WORKSPACE_MORE_FEATURES.getRoute(policyID));
+                            }}
```

Comment:
What about moving this to a separate function?

Reply:
Done - 5107292ed86cfb912ea504e143073a178bd3ee3b.